submissions/Harnytskyi/task_js_dom/js/script.js

```diff
@@ -0,0 +1,101 @@
+        img: "img/longcat.jpg"
+    },
+    {
+        title: "Grumpy Cat",
+        about: "Grumpy Cat is the nickname given to Tardar Sauce, a snowshoe cat that rose to online fame after several pictures of her annoyed facial expressions were posted to Reddit in late September 2012. Grumpy Cat was born in Morristown, Arizona on April 4th, 2012 to her owner Tabatha Bundesen. The original photos of Grumpy Cat were posted to the /r/pics subreddit by Bundesen's brother Bryan on September 23rd, 2012 (shown below).",
+        img: "img/grumpy-cat.jpg"
+    },
+    {
+        title: "Crying Cat",
+        about: "Crying Cat, also known as Schmuserkadser, refers to a series of photoshopped images of cats with teary, glassy eyes to appear as though they are sad. The earliest known usage of the Crying Cat was published by an anonymous Meme Generator user on June 11th, 2014 (shown below, left). The image is a photoshopped version of Serious Cat, due to the fact that the background of the image and ears and color of the cat are identical (shown below, right). Within four years, the image has generated more than 925 images.",
+        img: "img/crying-cat.jpg"
+    },
+    {
+        title: "Whoosh",
+        about: "Whoosht, known in Russia as вжух, is an image of a cat crudely dressed as a wizard that casts unfortunate spells. It grew popular on Russian websites such as Numl in December of 2016. The image of the cat is taken from a post on justcuteanimals.com (shown below, left). It later gained some traction as \"Wizard Cat\" on me_irl following a post from April 2016 that referred to it as the High Wizard Cat (shown below, right).",
+        img: "img/whoosh-cat.jpg"
+    }
+]
+
+const nav_ul = document.querySelector(".nav_ul");
+const aside = document.querySelector("aside");
+const main = document.querySelector("main");
+let openbtn = document.querySelector(".nav-open-button");
+let navButtonListArray;
+let selectedNav;
+
+function createNavButtons() {
+    let navLi;
+    let navButtonList = document.createDocumentFragment();
+    let navButtonItem;
+
+    nav_ul.innerHTML = "";
+    data.forEach(function (element) {
+        navLi = document.createElement("li");
+        navButtonItem = document.createElement("button");
+        navButtonItem.classList.add("nav-button");
+        navButtonItem.innerHTML = element.title;
+        navLi.append(navButtonItem);
+        navButtonList.append(navLi);
+    });
+    nav_ul.append(navButtonList);
+    navButtonListArray = Array.from(document.querySelectorAll(".nav-button"));
+}
+
+function displayMain({ target }) {
+    if (target.classList == 'nav-button') {
+        highlight(target);
+    }
+    const selectedButtonIndex = navButtonListArray.indexOf(target);
+    let element;
```

Comment:
I'd recommend using separate variables for different elements, because it would be complicated to follow only one variable and check appropriate elements